### DISCUSSION TITLE
Do you accept Pull Requests for reserved.ts?

### DISCUSSION THREAD

**Discussion by arayaryoma** [bluesky-social/atproto#1269] [Dev Debugging] [answered]
Hello, I'm a web developer from Japan. Upon reading the content of [reserved.ts](https://github.com/bluesky-social/atproto/blob/4db7aeca574871ea3f8365635e742fc4c4bdfbf8/packages/identifier/src/reserved.ts#L994-L1026), I noticed a bias in the reserved IDs for Japan. It doesn't seem like they're simply chosen based on the number of Twitter followers in Japan or selected from news organizations, but also include influencers.

On the other hand, I noticed that there're Twitter accounts with more followers than the listed accounts are not included. For instance, https://twitter.com/livedoornews, https://twitter.com/McDonaldsJapan, and https://twitter.com/Suzu_Mg among others.

Are there any policies for the list? Also, do you accept pull requests to update the list?

**Comment by bnewbold** [comment, marked as answer]
We are not looking to expand this list in an organized way, and the contents are somewhat arbitrary. There was a wave of impersonation with prominent Japanese handles, which upset some users, and we reserved a batch of top Japanese twitter handles in response. We don't have plans to expand that list, or pre-reserve handles from other regions.

Our general policy is that handles are first-come, first-served: a person or organization which had a username on a previous platform does not automatically get that handle on `*.bsky.social`. We encourage prominent individuals and institutions to use domain handles to prove their identity. For example, `mcdonalds.co.jp`. Adding additional handles to the reserved list runs somewhat counter to that policy, in that it marks some handles for special treatment, which isn't very fair to other folks.

All that being said, if there is an urgent situation or need, we could be flexible.

**Comment by arayaryoma** [comment]
@bnewbold Thank you for answering!! That makes sense!
